### PR TITLE
Fixes admin rewards page editting

### PR DESCRIPTION
**Pull request by yariplus**
Fixes https://github.com/NodeBB/nodebb-rewards-essentials/issues/6
Fixes https://github.com/NodeBB/nodebb-rewards-essentials/issues/11

### CODE REVIEW

File: src/views/admin/extend/rewards.tpl [hunk 1]
```diff
@@ -27,9 +27,9 @@
 							<div class="well inline-block">
 								<label for="condition">[[admin/extend/rewards:condition-then]]</label><br />
 								<select name="rid" data-selected="{active.rid}">
-									<!-- BEGIN rewards -->
+									<!-- BEGIN ../../rewards -->
```
Comment (pitaj) [inline src/views/admin/extend/rewards.tpl]: What about `conditions` and `conditionals`? Do they need to be modified the same way?

Comment (yariplus) [reply src/views/admin/extend/rewards.tpl]: No, those don't refer to scoped variables like active.rewards does.

File: src/views/admin/extend/rewards.tpl [hunk 2]
```diff
@@ -27,9 +27,9 @@
...
 									<option value="{rewards.rid}">{rewards.name}</option>
-									<!-- END rewards -->
+									<!-- END ../../rewards -->
```
Comment (julianlam) [inline src/views/admin/extend/rewards.tpl]: For future reference, this can be `<!-- END -->` now as well :smile:

**Comment by julianlam** [comment]
@yariplus I rebased this onto `develop` as v1.7.0 is launching tomorrow. Thanks!

File: src/views/admin/extend/rewards.tpl (hunk 2 above)
Comment (yariplus) [reply src/views/admin/extend/rewards.tpl]: I noticed! It's nice that you can still put anything in there though, helps with readability.

**Comment by wellenreiter01** [comment]
I applied the fix manually to my installation of nodebb.
The group and conditional pull downs are populated the 'then' pulldown is populated when adding a rewards. but is empty for existing rewards

**Pull request merged**